modules/Rock.cmake

```diff
@@ -141,6 +141,11 @@ at toplevel, like this:\
         enable_testing()
     endif()
 
+    option(ROCK_CXX_GCOV_ENABLED "Compile with coverage generation (enabled by default if ROCK_TEST_GCOV_GENERATION is set)" ON)
+    if (ROCK_CXX_GCOV_ENABLED)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --coverage -fprofile-abs-path")
```

Comment:
Do not change CMAKE_CXX_FLAGS. Add one of the commands to update compile options.